src/js/wasm_helper.js

```diff
@@ -8,7 +21,7 @@ const {
   BBS_SIGNATURES_MODES,
 } = require("./util");
 
-module.exports = function getWasmInstance() {
+module.exports = async function () {
```

Comment:
I assume this is switched to an anonymous function cause we're never calling it outside when we're exporting. Can you confirm?

Reply:
Yeap its basically the default export now so it can be anonymous